dataprofiler/labelers/data_processing.py

```diff
@@ -1416,15 +1417,15 @@ def get_parameters(self, param_list: List[str] = None) -> Dict:
         return params
 
     def convert_to_unstructured_format(
-        self, data: np.ndarray, labels: Optional[List[str]]
+        self, data: np.ndarray, labels: Optional[Union[List[str], npt.NDArray[np.str_]]]
     ) -> Tuple[str, Optional[List[Tuple[int, int, str]]]]:
         """
         Convert data samples list to StructCharPreprocessor required input data format.
 
         :param data: list of strings
         :type data: numpy.ndarray
         :param labels: labels for each input character
-        :type labels: list
+        :type labels: Union[list, numpy.ndarray]
```

Comment:
potentially we should include the `optional` and `str` here too, @Sanketh7 ?

`Optional[Union[List[str], npt.NDArray[np.str_]]]`

Reply:
updated